src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,388 @@
+
+### Create a replication user and password
+
+The standby cluster connects to the primary cluster's system interface using an identity with the `REPLICATION` privilege. Connect to the primary cluster's system interface and create a user with a password:
+
+1. From the primary's system interface SQL shell, create a user and password:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH PASSWORD '{your password}';
+    ~~~
+
+1. Grant the [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) to your user: {% comment %}Link to detail on replication system privilege{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT SYSTEM REPLICATION TO {your username};
+    ~~~
+
+    If you need to change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}).
+
+### Connect to the primary virtual cluster (optional)
+
+1. If you would like to run a sample workload on the primary's application virtual cluster, open a new terminal window and use [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}) to run the workload.
+
+    For example, to initiate the [`movr`]({% link {{ page.version.version }}/movr.md %}) workload:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload init movr "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
+    ~~~
+
+    {% include {{ page.version.version }}/connect/cockroach-workload-parameters.md %} As a result, for the example in this tutorial, you will need:
+    - `options=-ccluster=application`
+    - `sslmode=verify-full`
+    - `sslrootcert={path}/certs/ca.crt`: the path to the CA certifcate.
+    - `sslcert={path}/certs/client.root.crt`: the path to the client certificate.
+    - `sslkey={path}/certs/client.root.key`: the path to the client private key.
+
+    For additional detail on the standard CockroachDB connection parameters, refer to [Client Connection Parameters]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url).
+
+1. Run the `movr` workload for a set duration using the same connection string:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload run movr --duration=5m "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
+    ~~~
+
+1. To connect to the primary's application virtual cluster, use the `ccluster=application` parameter:
```

Comment:
```suggestion
1. To connect to the primary cluster's application virtual cluster, use the `ccluster=application` parameter:
```
I think it might be worthwhile to always refer to "primary cluster" and standby cluster" rather than implying them.

Reply:
I have changed up where I have found these